### PR TITLE
feat : delete the heading Sponsors

### LINKED ISSUE #4888: Public Event Page: Add menu tab "Sponsors"
If an organizer has activated "Sponsors" in the event creation wizard and added sponsors, please show a tab "Sponsors" on the left hand-side of the public page.

1. Implement the sponsors menu tab above the menu item "Organizers".
2. Change the heading of the "Sponsors" section to match the heading level and style of "Organizers".

Screenshot from 2020-08-27 19-04-10

### PR DESCRIPTION
**Pull request by indavarapuaneesh**
Fixes #4888  

#### Short description of what this resolves:
Only Supported By heading is displayed. The Sponsors heading is not displayed.

#### Changes proposed in this pull request:
deleted the header Sponsors in sponsor-list.hbs 

#### This is the screenshot in my local machine
Selection_005

#### Checklist

- [X] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [X] My branch is up-to-date with the Upstream `development` branch.
- [X] The acceptance, integration, unit tests and linter pass locally with my changes 
- [X] I have added tests that prove my fix is effective or that my feature works
- [X] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by iamareebjamal** [comment]
Now this'll remove Sponsors text from other places where it's used

**Comment by indavarapuaneesh** [comment]
> Now this'll remove Sponsors text from other places where it's used

In the file index.hbs the sponsors are being passed to the sponsors-list.js which in turn has a template sponsors-list.hbs, sponsors-list.hbs is where the sponsors are being displayed. As it is mentioned, deleting the heading won't do any good for other components. So for displaying sponsors, can I add redundant code that is already present in sponsors-list.hbs?. If not, any hints that could help solve this issue.

File: app/templates/components/public/sponsor-list.hbs [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<h3 class="ui header">{{t 'Sponsors'}}</h3>
+<h3 class="ui header"/>
```
Comment (maze-runnar) [inline app/templates/components/public/sponsor-list.hbs:1]: why keeping this line, remove it.

Comment (indavarapuaneesh) [reply app/templates/components/public/sponsor-list.hbs:1]: if I remove this line, then any other components using this component will be affected. This is what I understood from the previous comment by @iamareebjamal.
If that's incorrect then I can delete the whole <h3/> line and make a pull request.

**Comment by iamareebjamal** [comment]
Please pass a prop `@hideHeader` and pass it as true in public section so that it only hides the header in public section

**Pull request closed without merging**